packages/web-app-files/src/router/trash.ts

```diff
@@ -0,0 +1,46 @@
+import { RouteComponents } from './router'
+import { Location, RouteConfig } from 'vue-router'
+import { createLocation, $gettext, isLocationActiveDirector } from './utils'
+
+type trashTypes = 'files-trash-personal' | 'files-trash-spaces-project'
+
+export const createLocationTrash = (name: trashTypes, location = {}): Location =>
+  createLocation(name, location)
+
+export const locationTrashPersonal = createLocationTrash('files-trash-personal')
+export const locationTrashProject = createLocationTrash('files-trash-spaces-project')
+
+export const isLocationTrashActive = isLocationActiveDirector<trashTypes>(
+  locationTrashPersonal,
+  locationTrashProject
+)
+
+export const buildRoutes = (components: RouteComponents): RouteConfig[] => [
+  {
+    path: '/trash',
+    redirect: (to) => createLocationTrash('files-trash-personal', to),
+    component: components.App,
+    children: [
+      {
+        name: locationTrashPersonal.name,
+        path: 'personal',
+        component: components.Trashbin,
+        meta: {
+          hideFilelistActions: true,
+          hasBulkActions: true,
+          title: $gettext('Deleted files')
+        }
+      },
+      {
+        name: locationTrashProject.name,
+        path: 'spaces/projects/:storageId',
+        component: components.Spaces.Trashbin,
+        meta: {
+          hideFilelistActions: true,
+          hasBulkActions: true,
+          title: $gettext('Deleted files')
```

Comment:
```suggestion
          title: $gettext('Deleted files for/in the current space')
```

Reply:
Good hint let me talk about this with @tbsbdr